zrml/lsd-lmsr/src/constants.rs

```diff
@@ -0,0 +1,36 @@
+use crate::types::Timespan;
+use substrate_fixed::{types::extra::U24, types::extra::U32, FixedU32};
+
+// --- Default configuration for EmaConfig struct ---
+/// Default short EMA period (in seconds)
+/// One hour.
+pub const EMA_SHORT: Timespan = Timespan::Hours(1);
+
+/// Default long EMA period (in seconds)
+/// Six hours.
+pub const EMA_LONG: Timespan = Timespan::Hours(6);
+
+/// Default smoothing factor for EMA calculation
+/// 2.0
+pub const SMOOTHING: FixedU32<U24> = <FixedU32<U24>>::from_bits(0x0200_0000);
+
+// --- Default configuration for FeeSigmoidConfig struct ---
+/// Initial fee f
+/// 0.03
+pub const INITIAL_FEE: FixedU32<U32> = <FixedU32<U32>>::from_bits(0x07AE_1478);
```

Comment:
`INITIAL_FEE` is actually `0.0299999993`. Is this intended?

Reply:
> `INITIAL_FEE` is actually `0.0299999993`. Is this intended?

No, assigned the correct value to it and some other constants now. Thanks!